extensions/wikia/CreateNewWiki/maintenance/dumpStarters.php

```diff
@@ -119,18 +123,29 @@ private function generateSqlDump( $filename, $starter ) {
 	 * @throws DumpStartersException
 	 */
 	private function storeDump( $filename, $dest ) {
+		global $wgWikiaEnvironment;
+		global $wgGcsConfig;
+
+		$bucketName = self::BUCKET_NAME_PROD;
+
+		if ( $wgWikiaEnvironment == 'dev' ) {
+			$bucketName = self::BUCKET_NAME_DEV;
+		}
+
 		$this->output( sprintf( " \n\t[%s / %.2f kB]", $dest, filesize( $filename ) / 1024 ) );
 
-		$swift = \Wikia\CreateNewWiki\Starters::getStarterDumpStorage();
-		$res = $swift->store(
-			$filename,
-			$dest,
-			[],
-			self::DUMP_MIME_TYPE
-		);
+		$storage = new StorageClient( [ 'keyFile' => $wgGcsConfig['gcsCredentials'] ] );
+		$content = fopen( $filename, 'r' );
+		$bucket = $storage->bucket( $bucketName );
+		$gcsPath = sprintf( 'app/%s', $dest );
```

Comment:
Any reason we're using `sprintf` and not `"app/{$dest}"` instead?

Reply:
No reason other than to remain consistent with existing file. Plus, evidently 10x faster --> https://stackoverflow.com/questions/4195937/what-are-some-good-php-performance-tips/48200807#48200807